### ISSUE TITLE
Include softmax_act in model.predict in params of metrics

### ISSUE THREAD

**Issue by annahedstroem** [understandable-machine-intelligence-lab/Quantus#98]
Allow the user to specify `softmax_act` to `True` or `False` (for softmax or logit application) that is used in `model.predict` by adding it to `kwargs` of all the metrics

**Comment by annahedstroem** [comment]
Fixed here @https://github.com/understandable-machine-intelligence-lab/Quantus/pull/102 thanks @Wickstrom

**Comment by annahedstroem** [comment]
Fixed here @https://github.com/understandable-machine-intelligence-lab/Quantus/pull/102 thanks @Wickstrom
